src/ethoscope/core/variables.py

```diff
@@ -130,4 +130,11 @@ def _get_absolute_value(self, roi):
         out += oy
         return YPosVariable(out)
 
+class SubRoiValueVariable(BaseIntVariable):
```

Comment:
Make a new base class: `BaseTinyIntVariable`, and derive `SubRoiValueVariable` from it.
Also, verify that you can store a *unsigned* (8b)int inside. `"TINYINT UNSIGNED"` ? check that SQLite supports it (not only MySQL).